example/storybook/stories/TrackTile/Indicator.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react-native';
+import Indicator, {
+  aliases,
+} from '../../../../src/components/TrackTile/icons/indicator';
+import * as Icons from '@lifeomic/chromicons-native';
+import { kebabCase, invert } from 'lodash';
```

Comment:
Please file this comment under 'bikeShedding' and 'thingsThatDontMatter', but I much prefer this style of import over the `_kebabCase` or `_invert` style that is also popular.  👍